cypress/e2e/filter-feature.cy.ts

```diff
@@ -0,0 +1,60 @@
+describe('tests the enabling of different features', () => {
+  it('passes', () => {
+
+    //checks contract component
+    cy.visit(Cypress.env('blankUrl')).then(() => {
+        cy.window().invoke('initializeAstral', {enabledFeatures:["Contrast"]})
+    });
+    cy.waitForResource('main.js');
+    cy.document().find('astral-contrast');
+
+
+    //checks screen reader component
+    cy.visit(Cypress.env('blankUrl')).then(() => {
+      cy.window().invoke('initializeAstral', {enabledFeatures:["Screen Reader"]})
+    });
+    cy.waitForResource('main.js');
+    cy.document().find('astral-screen-reader');
```

Comment:
here we should check that only Screen Reader is present on screen, but nothing else.